README.rst

```diff
@@ -18,11 +18,42 @@
 PyProf - PyTorch Profiling tool
 ===============================
 
-    **NOTE: You are currently on the r21.03 branch which tracks stabilization 
-    towards the release. This branch is not usable during stabilization.**
-
 .. overview-begin-marker-do-not-remove
 
+PyProf is a tool that profiles and analyzes the GPU performance of PyTorch
+models. PyProf aggregates kernel performance from `Nsight Systems
+<https://developer.nvidia.com/nsight-systems>`_ or `NvProf
+<https://developer.nvidia.com/nvidia-visual-profiler>`_ and provides the
+following additional features:
+
+What's New in 3.9.0
+-------------------
+
+* Nothing new
```

Comment:
Do not use "Nothing new". If nothing has changed, just add "Compatibility NGC 21.03 release"